src/features/info/news/apiSlice.ts

```diff
@@ -1,6 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2023 BMW Group AG
- * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
why are you changing the year? the file was still introduced in 2023.
```suggestion
 * Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Reply:
@evegufy it is a new file. So I believe this change is valid

Reply:
@manojava-gk it's shown a changed file not a new file
why to think it's new?

Reply:
content is totally changed here. from ```api.ts``` to ```apiSlice.ts```  looks like rename tough, I will consider it as a new file.

Reply:
@evegufy it is a new file I changed the year to 2025 now.